tICA/scripts/ComputeTICAFeatures.m

```diff
@@ -317,7 +317,7 @@
 %Find strongest individual subjects for components
 [~, I]=max(TCSVARS);
 for i=1:size(TCSVARS,2)
-    %if ~isfile([OutputFolder '/tICA' num2str(i,'%02.f') '_SS.sdseries.nii']) && ~isfile([OutputFolder '/All' num2str(i,'%02.f') '_SS.sdseries.nii'])
+    if ~isfile([OutputFolder '/tICA' num2str(i,'%02.f') '_SS.sdseries.nii']) && ~isfile([OutputFolder '/All' num2str(i,'%02.f') '_SS.sdseries.nii'])
```

Comment:
If someone reruns tICA from an earlier step with some parameter change, rather than rerunning only the compute features step, what happens?  I don't like the logic of "this file exists" being assumed to imply "it is up to date with the settings and data being used".  The previous logic below of "if the file exists, don't save it, even though we just computed it" is similarly worrisome.

Reply:
I see your point. My current plan is to remove all the `isfile` check when this pipeline is officially released (strategy 1) or we introduce a default flag `-rerun` to control whether we want to skip the file check or not (strategy 2). Which one do you think is better?

Reply:
This revision won't happen in this PR though, but I think we can keep the current script this way for the fix only...

Reply:
It is being included in the qunex container as-is, I think we should set it up now to work the way we want it to work for users long term.

I think by default it should regenerate every file it creates on the first run, rather than doing something different when rerun.  I'm not clear on why there should be an option to do otherwise (when you are testing something for which it is valid to skip things, you can edit the code and run interpreted, and never commit those edits to the repo).

Reply:
I see. That is said, you would prefer to remove all the file check operations in the ComputeFeature.m in this PR?

Reply:
Might as well do it now, rather than doing another recompile later, or forgetting to change it at all.